libraries/chain/include/eosio/chain/exceptions.hpp

```diff
@@ -681,4 +681,13 @@ namespace eosio { namespace chain {
    FC_DECLARE_DERIVED_EXCEPTION( state_history_exception,    chain_exception,
                                  3280000, "State history exception" )
 
+   FC_DECLARE_DERIVED_EXCEPTION( ssl_exception, chain_exception,
+                                 3290000, "SSL exception")
+
+      FC_DECLARE_DERIVED_EXCEPTION( ssl_incomplete_configuration, ssl_exception,
```

Comment:
make indentation on this line and following match the ones above

Reply:
the one above is root class exception